2020/13xxx/CVE-2020-13671.json

```diff
@@ -1,18 +1,87 @@
 {
```

Comment:
{} is not allowed for {'vendor': {'vendor_data': [{'vendor_name': 'Drupal ', 'product': {'product_data': [{'product_name': 'Drupal Core', 'version': {'version_data': [{'version_name': '9.0', 'version_affected': '<', 'version_value': '9.0.8', 'platform': ''}, {'version_name': '8.9', 'version_affected': '<', 'version_value': '8.9.9', 'platform': ''}, {'version_name': '8.8', 'version_affected': '<', 'version_value': '8.8.11', 'platform': ''}, {'version_name': '7', 'version_affected': '<', 'version_value': '7.74', 'platform': ''}]}}]}}]}}